shogun-boot/src/main/java/de/terrestris/shogun/boot/config/BootWebSecurityConfig.java

```diff
@@ -35,6 +35,11 @@ protected void customHttpConfiguration(HttpSecurity http) throws Exception {
                     "/index.html"
                 )
                     .permitAll()
+                .antMatchers(
+                    "/swagger-ui.html",
```

Comment:
The `swagger-ui.html` could IMO also be accessible for all, the actual requests it can send would still be restricted anyway.

Reply:
Agreed, fixed in 8c14b0a.